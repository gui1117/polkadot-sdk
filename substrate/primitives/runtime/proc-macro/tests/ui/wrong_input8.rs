// This file is part of Substrate.

// Copyright (C) Parity Technologies (UK) Ltd.
// SPDX-License-Identifier: Apache-2.0

// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
// 	http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

use sp_runtime_proc_macro::with_features_try_runtime_and_runtime_benchmarks;

#[with_features_try_runtime_and_runtime_benchmarks(feature("std"))]
fn foo() {}

fn main() {
}
