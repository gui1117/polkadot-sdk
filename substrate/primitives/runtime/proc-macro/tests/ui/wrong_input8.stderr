error: unexpected token
  --> tests/ui/wrong_input8.rs:20:59
   |
20 | #[with_features_try_runtime_and_runtime_benchmarks(feature("std"))]
   |                                                           ^^^^^^^
